tests/base.py

```diff
@@ -81,6 +114,29 @@ def assertEqualJsonBom(self, a: str, b: str) -> None:
 
 class BaseXmlTestCase(TestCase):
 
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.FIXTURES_DIRECTORY = path.join(path.dirname(__file__), 'fixtures')
+
+    def assertValidAgainstSchema(self, bom_xml: str, schema_version: SchemaVersion) -> None:
+        xsd_fn = os.path.join(schema_directory, f'bom-{schema_version.name.replace("_", ".").replace("V", "")}.xsd')
+        with open(xsd_fn) as xsd_fd:
+            xsd_doc = etree.parse(xsd_fd)
+
+        xml_schema = etree.XMLSchema(xsd_doc)
+        schema_validates = False
+        try:
+            schema_validates = xml_schema.validate(etree.parse(io.BytesIO(bytes(bom_xml, 'ascii'))))
+        except DocumentInvalid as e:
```

Comment:
whyyyyy? 
why catch the exception, if this exception is clearly a reason that this test would fail?